### ISSUE TITLE
wowwiki.wikia.com

### ISSUE THREAD

**Issue by byaka** [reek/anti-adblock-killer#3665]
http://wowwiki.wikia.com/wiki/World_of_Warcraft_API
http://imgbox.com/tK69uyiN
Google Chrome
Tampermonkey
Ublock Origin
Alot

**Comment by uBlock-user** [comment]
Follow the [template](https://github.com/reek/anti-adblock-killer/blob/master/CONTRIBUTING.md#require) and edit your post accordingly.

**Comment by Perritu** [comment]
Hi. I was curious about how to kill it's adblock-bypasser, and once again, 

> The chain is broken by the weakest link

# TL;DR:
This code solves it (also fewly obfuscated)
```
(function(dti){
    var dy=dti.apply(this,['love']);
})((function(sim,aak){
    return function(){
        sim('.WikiaSiteWrapper').parent().addClass(aak);
        sim('body > *:not(.'+aak+')').remove();
    }

    var aakcss=document.createElement('style');
    aakcss.innerHTML=".WikiaTopAds + div {display: none !important;position: fixed !important;top: -500vh !important;}";
    document.body.appendChild(aakcss);
}(jQuery,"aak"+Math.floor(Math.random(0,1)*parseInt("aakAAKaakAAKaak",36)).toString(36))));
```
# Explain
```
(function(dti){
    var dy=dti.apply(this,['love']);
})
```
Just an dummy function to do 2 thing: 1. obfuscate the ejecution and 2. anonymize the function itself (double anonymize).

```
sim('.WikiaSiteWrapper').parent().addClass(aak);
sim('body > *:not(.'+aak+')').remove();
```
This is the magic.
`sim` is an alias to jQuery (obfuscation?).
`'.WikiaSiteWrapper'` is the wikia's indicator (which reveals the true span in all these ads containers on body DOM).
`aak` is a random alphanumeric identificator added to the "true" container to avoid it's destruction (MWAHAHAHA) <- Malevolent laugh
```
var aakcss=document.createElement('style');
    aakcss.innerHTML=".WikiaTopAds + div {display: none !important;position: fixed !important;top: -500vh !important;}";
    document.body.appendChild(aakcss);
```
this is just to remove the white space created by an ad container on the header (revealed by the "weakest link", Remember?). The CSS code (in a more readable way) is:
```
.WikiaTopAds + div {
    display: none !important;
    position: fixed !important;
    top: -500vh !important;
}
```
Hope this help this wonderfull tool :smile:

**Comment by uBlock-user** [comment]
or you could just add `abtesting$script` in your blocker and be done with it.

**Comment by byaka** [comment]
@uBlock-user awesome solution, thx!
